sherpa/astro/flux.py

```diff
@@ -370,14 +375,17 @@ def decompose(mdl):
 def sample_flux(fit, data, src,
                 method=calc_energy_flux, correlated=False,
                 num=1, lo=None, hi=None, numcores=None, samples=None,
-                clip='hard'):
+                clip='hard', rng=None):
     """Calculate model fluxes from a sample of parameter values.
 
     Draw parameter values from a normal distribution and then calculate
     the model flux for each set of parameter values. The values are
     drawn from normal distributions, and the distributions can either
     be independent or have correlations between the parameters.
 
+    .. versionchanged:: 4.15.1
```

Comment:
Dang it. Need to update the version number,

Reply:
The commit message for this PR also has a typo, so I must try to remember to change that too...